assets/js/index.js

```diff
@@ -1,9 +1,17 @@
+---
+---
+
 // Generic App Class
 
 class App {
   constructor(container) {
     this._init(container);
     this._bindEvents();
+    this.setMobileNavOnMobileView.bind(this);
```

Comment:
`this.setMobileNavOnMobileView = this.setMobileNavOnMobileView.bind(this)`
Then we don't need to create arrow functions for calling all the methods